modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1534,6 +1580,96 @@ Enables the translation of topic data into Iceberg tables. Setting `iceberg_enab
 
 ---
 
+=== iceberg_rest_catalog_endpoint
+
+URL of Iceberg REST catalog endpoint.
```

Comment:
@andrwng is this meant to be set together with / as a pair with `iceberg_catalog_base_location`?